modules/nlu/src/views/full/common/TrainNow.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from '@blueprintjs/core'
+import { NLUApi } from 'api'
+import React, { FC, useState } from 'react'
+
+const TrainNow: FC<{ api: NLUApi }> = ({ api }) => {
+  const [training, setTraining] = useState(false)
+
+  const train = async () => {
+    setTraining(true)
+    await api.train()
+    setTraining(false)
+  }
+  return (
+    <Button onClick={() => train()} disabled={training}>
+      Train now
+    </Button>
```

Comment:
If you refresh the page, you can click the button again. What do you think of querying the server to know if a training is in progress, and disable the button? 

Also, if we have the capacity to cancel the training, it could be nice to have "Cancel training" while training is in progress

Reply:
@allardy done